src/debugger.ts

```diff
@@ -14,13 +14,13 @@ export class Debugger
 {
   private debugProcess?: ChildProcessWithoutNullStreams;
   private readonly console = vscode.debug.activeDebugConsole;
-  private readonly currentActiveWorkspace: () => Workspace | undefined;
+  private readonly getWorkspace: (uri: vscode.Uri) => Workspace | undefined;
 
   constructor(
     context: vscode.ExtensionContext,
-    currentActiveWorkspace: () => Workspace | undefined,
+    getWorkspace: (uri: vscode.Uri) => Workspace | undefined,
```

Comment:
(nit) The `get` prefix may suggest this is an accessor for `workspace`... could this instead be something like `resolveWorkspace`?